lib/Controller/ChatController.php

```diff
@@ -76,6 +77,39 @@ public function __construct($appName,
 	}
 
 	/**
+	 * Returns the Room for the current user.
+	 *
+	 * If the user is currently not joined to a room then the room with the
+	 * given token is returned (provided that the current user is a participant
+	 * of that room).
+	 *
+	 * @param string $token the token for the Room.
+	 * @return \OCA\Spreed\Room|null the Room, or null if none was found.
+	 */
+	private function getRoom($token) {
```

Comment:
little remark for the future, please use protected rather then private, makes testing and other things a lot easier

Reply:
I used _private_ because this class is not meant to be extended and because, in general, I prefer to not change a design just to make testing easier (and if I am not mistaken this change would help only to be able to mock the method, as you could invoke it to explicitly test it by using reflection).

Having said that, I can change it to _protected_ if you wish ;-)